user-guide/Advanced_Modules/DOM/DOM_actions.md

```diff
@@ -163,7 +163,7 @@ An `ActionDefinition` can only be executed when a pre-defined condition is met.
 
 ## Defining buttons
 
-The `DomBehaviorDefinition` also contains a list of `IDomButtonDefinitions`. These can be used to have one or more buttons shown in the UI. These buttons can be linked to one or more actions that will be executed when they are clicked.
+The `DomBehaviorDefinition` also contains a list of `IDomButtonDefinitions`. These can be used to have one or more buttons shown in the UI. These buttons can be linked to an action that will be executed when they are clicked.
```

Comment:
Could be changed from " These buttons can be linked to an action" to " Each button can be linked to one action"

Reply:
Done, thanks.